src/old/v1/image/index.tsx

```diff
@@ -15,14 +15,12 @@ export interface ImageTheme {
   padding?: string
 }
 
-export interface ImageProps {
+export type ImageProps = React.PropsWithChildren<{
   customStyle?: ImageTheme
   id?: string
-  children?: HTMLSourceElement
```

Comment:
I think `children?: HTMLSourceElement | HTMLSourceElement[]` would be better than `React.PropsWithChildren` as they're the only children the picture element can take, right?

Reply:
Oh, just noticed this is used for the `figure` too. Ah well :laughing: